x/qgb/v1/consts.go

```diff
@@ -0,0 +1,5 @@
+package v1
+
+// SignificantPowerDifferenceThreshold the threshold of change in the validator set power
+// that would need the creation of a new valset request.
+const SignificantPowerDifferenceThreshold = 0.005
```

Comment:
I'm not really sure how we want to organize all of these changes for main/mainnet. This was just one exploratory option.

Reply:
Currently, in the QGB state machine, a validator set change is triggered if more than 5% of the total staking power has changed since the last valset.
The issue is that in BSR, all validators have 1% stake, and there aren't any changes to the validator set. Thus, if they change their EVM addresses, which should trigger a new validator set change if their stake is more than 5%, they will need to wait for another 4 validators to do that or else they won't be able to sign attestations

Reply:
That change reduces the threshold to 0.5% so that every validator that changes its EVM address (because they lost its private key), will be able to sign right after that

Reply:
Got it. Thanks for the clarification